### ISSUE TITLE
Contact field in graph, just like Alias and Color

### ISSUE THREAD

**Issue by AbelLykens** [lightningnetwork/lightning-rfc#823]
Just like we have `alias` and `color` for nodes, it would be nice to have a 64-character free text contact field in the spec.

Imagine setting:
```
[Application Options]
contact="forwarderaddress@protonmail.com"
```
If this is synced to the graph (again, like alias and color), it will be much easier to find contact information. If I see a peer is down, I can look up the contact details. Of course people would be free to fill in whatever they want: a twitter link, throwaway email account, web site, ... Let's not make that strict.

This works well for Tor nodes, too. They also have a contact field.

**Comment by AbelLykens** [comment]
Many different sites are now building this. For example https://amboss.space/ and https://1ml.com/ are now building their own database, which is fine, but I would rather have a graph-gossiped field.

Wonder what the rest thinks.

**Comment by rbndg** [comment]
Makes sense to me, Domain name register haves have the same feature as well

**Comment by kiz8** [comment]
had a situation recently where I really needed this information but it couldnt be found. would LOVE to see this happen.

**Comment by TheBlueMatt** [comment]
This seems like a very misguided feature. Normalizing identities tied to lightning nodes is incredibly problematic and may lead more to the idea that they are counterparties instead of just automated software with no relation to the payments being made (which is a much more real-world accurate interpretation).
